package.json

```diff
@@ -63,6 +63,8 @@
     "junk": "^3.1.0",
     "locate-path": "^5.0.0",
     "make-dir": "^3.1.0",
+    "merge-options": "^3.0.4",
+    "minimatch": "^3.0.4",
```

Comment:
`minimatch` is unmaintained.
Should we use a different library, e.g. https://github.com/micromatch/micromatch / https://github.com/mrmlnc/fast-glob ?

Reply:
I chose `minimatch` because `micromatch` is like three times larger. `fast-glob` uses `micromatch` itself. But you're right, the state of `minimatch` doesn't look good.

Reply:
We can maybe choose another library than both of those.
But in general, I think the fact it has been mostly unmaintained for 4 years might become a problem: bugs and security fixes are not being applied, new Node.js major releases might break, etc.

Reply:
You're right. I'll make the change.

Reply:
Also: we use `minimatch` in `@netlify/cache-utils`, `netlify-cli` (indirectly) and `@netlify/git-utils` (directly), so this would not change a difference in npm package size when `@netlify/zip-it-and-ship-it` is included in one of those. 
(Also this might provide with consistency)

Reply:
(Note: I meant `micromatch` above, editing my comment)

`minimatch` works :+1: